Lib/ast.py

```diff
@@ -1053,15 +1054,82 @@ def visit_AsyncWith(self, node):
         with self.block(extra=self.get_type_comment(node)):
             self.traverse(node.body)
 
+    def _str_literal_helper(
+        self, string, quote_types=("'", '"', '"""', "'''"), escape=""
+    ):
+        """Helper for writing string literals, minimizing escapes.
+        Returns (possible quote types, string literal to write).
```

Comment:
Even though it is implicitly said with parens, it would be nice to clarify the return value is a 'tuple' of those.